lib/main.dart

```diff
@@ -175,6 +182,7 @@ void main() async {
             ),
             Provider<CommonLinkBloc>(value: commonLinkBloc),
             Provider<MaintenanceCheckBloc>(value: maintenanceCheckBloc),
+            Provider<ArticleListBlocFactory>(value: articleListBlocFactory),
```

Comment:
Put in alphabetical order (bloc factories -> blocs)

Reply:
done